### ISSUE TITLE
error running app

### ISSUE THREAD

**Issue by alexeian96** [konsalext/Expected-goals-report-app#1]
error running app

**Comment by konsalext** [comment]
Yup thanks for letting me know , it was an deprecated package in requirements.txt file and removing it  fixed it.Tried it just now  and it is working fine , give it a try
